app/e2e/incomingCall.test.ts

```diff
@@ -51,6 +53,10 @@ describe('Incoming Call', () => {
     await device.reloadReactNative();
   });
 
+  afterEach(async () => {
+    await new Promise((resolve) => setTimeout(resolve, 10000));
```

Comment:
What is this for?

Reply:
Can we add a comment to this line so we don't forget?

Reply:
Sounds good. Hopefully it doesn't flake again 😛